indigo_app/static/javascript/indigo/grammars/slaw.js

```diff
@@ -46,6 +46,9 @@
       this.theme_def = {
         base: 'vs',
         inherit: true,
+        colors: {
+          'editor.foreground': '#000000'
+        },
```

Comment:
what's this about?

Reply:
Monaco editor changed its requirements and expects this to be there